src/hapi/pipelines/configs/population.yaml

```diff
@@ -1,6 +1,24 @@
 # Collector specific configuration
 commit_limit: 1000
 
+default:
+  keys:
+    - "scraper_national"
```

Comment:
What if this listed individual country scrapers instead of the admin level? That way AFG and NGA can use the long default list, and MLI can just be redefined

Reply:
To clarify you mean that only the declared country/admlvls will have the defaults inserted?

Reply:
Yes

Reply:
These changes have now been made